src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int i = new Random().nextInt(Color.values().length);
+        return Color.values()[i];
```

Comment:
we dont need to create Random instance each time when we call this method
```suggestion
private final Random random = new Random();

    public Color getRandomColor() {
        return Color.values()[random.nextInt(Color.values().length)];
```